inc/rssfeed.class.php

```diff
@@ -236,7 +236,7 @@ static function addVisibilityRestrict() {
       $sql = $it->getSql();
       $sql = preg_replace('/.*WHERE /', '', $sql);
 
-      return $sql;
+      return ['sql' => $sql, 'params' => $it->getParameters()];
```

Comment:
Signature update should be documented in changelog and PHPDoc should be updated.